packages/synapse-react-client/src/components/DirectDownload/DirectDownload.tsx

```diff
@@ -1,44 +1,38 @@
-import React, { useEffect, useState } from 'react'
+import React from 'react'
 import {
   BatchFileRequest,
-  FileEntity,
-  FileHandle,
+  ExternalFileHandle,
   FileHandleAssociateType,
   FileHandleAssociation,
 } from '@sage-bionetworks/synapse-types'
-import {
-  getFiles,
-  getFileResult,
-  getEntity,
-} from '../../synapse-client/SynapseClient'
+import { getFiles } from '../../synapse-client/SynapseClient'
 import IconSvg, { IconSvgProps } from '../IconSvg/IconSvg'
-import { useInView } from 'react-intersection-observer'
 import { useSynapseContext } from '../../utils/context/SynapseContext'
 import { TOOLTIP_DELAY_SHOW } from '../SynapseTable/SynapseTableConstants'
 import { UAParser } from 'ua-parser-js'
 import { Tooltip } from '@mui/material'
 import { implementsExternalFileHandleInterface } from '../../utils/types/IsType'
+import { useGetFileBatch } from '../../synapse-queries/file/useFiles'
 
 export type DirectFileDownloadProps = {
   associatedObjectId: string
-  entityVersionNumber?: string
```

Comment:
No longer accept `entityVersionNumber`. The FileHandleAssociation tuple of (file handle ID, object ID, object type) must now always be passed by the caller.